### ISSUE TITLE
Character clipping through world ceiling

### ISSUE THREAD

**Issue by ShaydenMac** [StrangeLoopGames/EcoIssues#7976]
If I'm putting down rock, dirt, etc. and I happen to be standing on it, as I do it pushes me up higher but if the ceiling is low, I clip through it and get stuck. The only way out is to pick the stuff I've dropped back up and I fall back down.

**Comment by BlazeGuy2** [comment]
I was sent falling through the world by placing hewnlogs with the hammer until i hit the ceiling.

**Comment by BlazeGuy2** [comment]
https://www.youtube.com/watch?v=JHwqrDCBQ7o&feature=youtu.be

**Comment by ShaydenMac** [comment]
Yeah, I didn't fall through the world, just clipped through the ceiling.
